src/stems/stabilizer.scad

```diff
@@ -0,0 +1,34 @@
+include <../functions.scad>
+
+// extra length to the vertical tine of the inside cherry cross
+// splits the stem into halves - allows easier fitment
+extra_vertical = 0.6;
+
+module inside_cherry_stabilizer_cross(slop) {
+  // inside cross
+  // translation purely for aesthetic purposes, to get rid of that awful lattice
+  translate([0,0,-0.005]) {
+    linear_extrude(height = $stem_throw) {
+      square(cherry_cross(slop, extra_vertical)[0], center=true);
+      square(cherry_cross(slop, extra_vertical)[1], center=true);
+    }
+  }
+
+  // Guides to assist insertion and mitigate first layer squishing
+  if ($cherry_bevel){
```

Comment:
no cherry bevel since it rips off the bed really easily and needs all the material it can get